client/components/TestQueue/AssignTesterDropdown/index.jsx

```diff
@@ -113,9 +113,33 @@ const AssignTesterDropdown = ({
     const clearAriaLiveRegion = () => {
         setAlertMessage('');
     };
+
+    const handleKeyDown = event => {
+        const { key } = event;
+        if (key.match(/[a-z]/)) {
```

Comment:
```suggestion
        if (key.match(/[0-9a-zA-Z]/)) {
```

While there are none currently tracked by the app, I've come to find out GitHub usernames can begin with a number, so just to be safe. Uppercase is there as well in case one is holding `Shift` or `CapsLock` is enabled. It seems to also align with browsers' native behavior for `<select>`